core/api/src/main/java/org/eclipse/sensinact/core/whiteboard/WhiteboardHandler.java

```diff
@@ -0,0 +1,20 @@
+/*********************************************************************
+* Copyright (c) 2024 Kentyou.
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which is available at https://www.eclipse.org/legal/epl-2.0/
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*   Thomas Calmant (Kentyou) - initial implementation
+**********************************************************************/
+package org.eclipse.sensinact.core.whiteboard;
+
+/**
+ * Common parent interface for whiteboard handlers
+ */
+public interface WhiteboardHandler<T> {
```

Comment:
I don't think that this needs generics - it also makes things difficult if you implement get/set/act in a single service as you can have different type hierarchies for the same interface with competing type parameters.